docs/swagger/api.yaml

```diff
@@ -87,13 +87,28 @@ paths:
             items:
               $ref: '#/definitions/PrintReceiptResponse'
         400:
+          description:
+            Connection to the device is established,
+            but a problem with the device occured and receipt was not printed.
+            Returns array of warnings
```

Comment:
oh I saw warnings here PrintReceiptResponse, but it is not returned by this error code?